### PR TITLE
Copy update /what-is-ros

### LINKED ISSUE #8429: [COPY UPDATE] /what-is-ros - "What is ROS"
Please update /what-is-ros - "What is ROS" from the [copy doc](https://docs.google.com/document/d/1yX2Ikcn1fgIKX3el8fKWQt7SGxSwY633TfW5V1lwUuc/edit?ts=5f7af517#heading=h.vbzgtki4500k)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Copy update on `/robotics/what-is-ros`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/robotics/what-is-ros
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against [copy doc](https://docs.google.com/document/d/1yX2Ikcn1fgIKX3el8fKWQt7SGxSwY633TfW5V1lwUuc/edit?ts=5f7af517#heading=h.vbzgtki4500k) and [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/8429)

## Issue / Card

Fixes #8429

### CODE REVIEW

**Comment by meralonso** [comment]
Hey @bethcollins92 this is looking good - just a couple of things

Could you move the last paragraph on the left and make it the first on the right column?

Screenshot 2020-10-21 at 13 52 21

At the bottom on the middle column the submit button is very close to the text - is it always like that or could you give it a bit more margin?

Screenshot 2020-10-21 at 13 53 31

**Comment by bethcollins92** [comment]
@meralonso Made those changes and ready for re-review

**Comment by meralonso** [comment]
LGTM 👍

File: templates/robotics/what-is-ros.html [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
+<section class="p-strip--suru-topped">
   <div class="row u-equal-height">
     <div class="col-8">
-      <h1>What is ROS?</h1>
-      <h2>ROS is powering the future of robotics in industry, in the enterprise and for developers</h2>
-      <p>At the offset, ROS chose Ubuntu as its primary supported platform and today it remains the platform of choice for the majority of ROS developers.</p>
-      <p>ROS is led by Open Robotics, similar to how Canonical supports Ubuntu, Open Robotics steers the ship but it is driven by the community.</p>
+      <h1>What is ROS</h1>
+      <p class="p-heading--four">ROS is powering the future of robotics in industry, in the enterprise and for developers</p>
+      <p>Ubuntu has been the primary platform for ROS from the very beginning, thanks to it's flexibility and user-friendliness.</p>
```
Comment (albertkol) [inline templates/robotics/what-is-ros.html:15]: Typo: "it's" -> "its"

File: templates/robotics/what-is-ros.html [hunk 2]
```diff
@@ -69,31 +68,30 @@ <h2>What is ROS?</h2>
...
         ) | safe
       }}
-    </div>
-    <div class="col-8">
       <h2>What is ROS 2?</h2>
       <p>ROS 2 is a complete re-design of the framework, tackling the shortcomings of the first generation, effectively bringing it to industry needs and standards. ROS 2 contains new sets of packages that can be installed alongside ROS 1 to ease migration to a more secure platform. ROS 2 takes advantage of new technologies and newly updated APIs asked for by the community.</p>
-      <p><a class="p-button--positive p-link--external" href="https://www.ros.org/contribute/">Contribute to ROS 2</a></p>
-      <p><a class="p-link--external" href="https://index.ros.org/doc/ros2/">Learn more about ROS 2 from the owners</a></p>
-    </div>
+      <p><a class="p-button--positive p-link--external" href="https://www.ros.org/contribute/">Get started with ROS 2</a></p>
```
Comment (albertkol) [inline templates/robotics/what-is-ros.html:90]: The doc says the url should point here: https://index.ros.org/doc/ros2/Tutorials/

**Review by albertkol — COMMENTED** [review]
Spotted two things:

**Review by albertkol — APPROVED** [review]
:+1:

**Pull request merged**